src/CsvTextEditor/Commands/EditFindReplaceCommandContainer.cs

```diff
@@ -0,0 +1,32 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="EditUndoCommandContainer.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace CsvTextEditor
+{
+    using Catel.IoC;
+    using Catel.MVVM;
+    using Orc.ProjectManagement;
+
+    public class EditFindReplaceCommandContainer : EditProjectCommandContainerBase
+    {
+        #region Constructors
+        public EditFindReplaceCommandContainer(ICommandManager commandManager, IProjectManager projectManager, IServiceLocator serviceLocator)
+            : base(Commands.Edit.FindReplace, commandManager, projectManager, serviceLocator)
+        {
+        }
+        #endregion
+
+        #region Methods
+        protected override void Execute(object parameter)
+        {
+            CsvTextEditorService.ShowFindReplaceDialog();
```

Comment:
I think of CsvTextEditorService as of control service. And control can open it's tool windows. It's like another way of editing, but i think i can separate findAndReplace and textEditor services